src/palace/manager/util/resources.py

```diff
@@ -0,0 +1,7 @@
+from importlib.abc import Traversable
+from importlib.resources import files
+
+
+def resources_dir(subdir: str) -> Traversable:
```

Comment:
We use this new function to load any content needed from the package. It uses `importlib` to get the path to the package. This handles the various cases where our code could end up in different package locations.